src/common.rkt

```diff
@@ -65,6 +65,13 @@
     (define row (for/list ([(hkey htype) (in-dict header)]) (dict-ref fields hkey)))
     (dict-set! rows key row)))
 
+(define/contract (table-remove! tbl key)
+  (-> (cons/c (listof (cons/c symbol? contract?)) (hash/c symbol? (listof any/c)))
+      symbol?
+      any)
```

Comment:
The return type here is `void?`, which also indicates that the return value is meaningless.